testgrid/transfigure.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+## Copyright 2019 Istio Authors
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##    http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.package main
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+root="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+export CONFIG_PATH="${root}/prow/config.yaml"
+export JOB_CONFIG_PATH="${root}/prow/cluster/jobs"
+export TESTGRID_CONFIG="${root}/testgrid/default.yaml"
+export ORG_REPO="istio"
+
+# Get the common instance of transfigure.sh and run it
+bash <(curl -s https://raw.githubusercontent.com/kubernetes/test-infra/master/testgrid/transfigure.sh) "$@"
```

Comment:
I think a better pattern for this will be to create a gcr.io/k8s-prow/transfigure or whatever image that runs this script

https://github.com/kubernetes/test-infra/blob/0492f1d6ebc23b10ebe8a756e81fcbacf85ac92a/prow/cmd/autobump/BUILD.bazel

https://github.com/istio/test-infra/blob/4cba48e9bbe4093f3770813f3c841866fc1c51e8/prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml#L128-L167

This is how @cjwagner did all the autobumper jobs. Maybe ask him to walk you through the design if reading through the above isn't obvious?